pkg/remote/remote.go

```diff
@@ -223,13 +249,18 @@ type GoGetter struct {
 func (g *GoGetter) Get(wd, src, dst string) error {
 	ctx := context.Background()
 
+	opts := []getter.ClientOption{}
+
+	// enable insecure features
+	opts = append(opts, getter.WithInsecure())
```

Comment:
This is probably for enabling insecure https access? Then I'd guess we'd like to use secure by default, and use insecure when requested (by user, or in test?)